indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -206,16 +208,20 @@ def _send_emails(self, form):
             template = get_template_module('events/registration/emails/custom_email.html',
                                            email_subject=form.subject.data, email_body=email_body)
             bcc = [session.user.email] if form.copy_for_sender.data else []
+            attachments = get_ticket_attachment(registration) if form.attach_ticket.data else None
             email = make_email(to_list=registration.email, cc_list=form.cc_addresses.data, bcc_list=bcc,
-                               from_address=form.from_address.data, template=template, html=True)
+                               from_address=form.from_address.data, template=template, html=True,
+                               attachments=attachments)
             send_email(email, self.event, 'Registration')
 
     def _process(self):
         tpl = get_template_module('events/registration/emails/custom_email_default.html')
         default_body = tpl.get_html_body()
         registration_ids = request.form.getlist('registration_id')
         form = EmailRegistrantsForm(body=default_body, regform=self.regform, registration_id=registration_ids,
-                                    recipients=[x.email for x in self.registrations])
+                                    recipients=[x.email for x in self.registrations], )
```

Comment:
is that change necessary?

Reply:
no, and it's rather misleading (one might first think you are using a tuple containing a list here). probably a leftover from some argument htat was there before